tests/integration/conftest.py

```diff
@@ -139,9 +141,15 @@ def __call__(self) -> ContextManager[ProtostarFixture]:
 
 @pytest.fixture(name="create_protostar_project", scope="module")
 def create_protostar_project_fixture(
+    request: FixtureRequest,
     session_mocker: MockerFixture,
     tmp_path_factory: TempPathFactory,
 ) -> CreateProtostarProjectFixture:
```

Comment:
After creating every cairo1 command we can probably just parametriaze the fixture there instead of doing it on the test level -> pytest.fixture(params=...).